tensorbay/client/dataset.py

```diff
@@ -178,6 +178,37 @@ def get_notes(self) -> Notes:
             self._client.open_api_do("GET", "notes", self._dataset_id, params=params).json()
         )
 
+    def update_draft(
+        self,
+        *,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        is_closed: bool = False,
```

Comment:
`update_draft` only used to update title and description,
we need another function `close_draft` to close draft